### DISCUSSION TITLE
What GPUs are ya'll fine MFs rockin?

### DISCUSSION THREAD

**Discussion by AryanEmbered** [YellowRoseCx/koboldcpp-rocm#48] [Polls]
I have a strong suspicion that this is a full on RX6600 gang here as it doesn't work with the official Rocm implementation on windows unlike other gpus while still being fast enough. cast your vote below

**Comment by PhialsBasement** [comment]
CASTRATED EDITION HAHAHAHAHA

too fuckin funny

**Comment by cloak1505** [comment]
You're supposed to include a catch all option for RX 6800 and above to get an idea of which percentage of the userbase actually has a "supported" GPU.

**Comment by RenanOSX** [comment]
I got my very simple RX580 8GB lol, I'm able to run stable difusion but I can't run Kobold that well

**Reply by SergeantBlyat** [reply]
Oh damn which webui do you use to run stable diffusion?

**Reply by RenanOSX** [reply]
Sorry for the late reply, I was using "stable-diffusion-webui-directml" with those tags: --use-directml --upcast-sampling --opt-sub-quad-attention --skip-torch-cuda-test --no-half-vae

The new version I couldn't make it work

**Comment by SpoiledCoding** [comment]
Rockin' a 7900XTX here. I'm incredibly grateful for the VRAM and the people doing the work on projects like this so we can actually use it without wrangling Linux or WSL. (I say nearing the brink of committing to the switch...)

**Reply by Tatalebuj** [reply]
Got the same, and had no idea about LLM's when I bought it.  I definitely regret going for the 24gb, but so happy compared to most others out there.  This entire project has been a "god-send" for me, lots of warm and happy thoughts for the good people doing all the actual work.

**Reply by monk-insane** [reply]
Got the 7800 XT Here, and loving the 16GB VRAM. Card is a beast. Best bang for buck. Would've liked more VRAM - but I would say the same if it had 32 GB lol.

**Comment by nielsijzerman** [comment]
AMD:   RX580 8GB, Durango 2 8GB, 2x HD7850 2GB, HD6970* 2GB, 2xHD7450 1GB, Xenos Xenon 512MB, HD5450 512MB **
Nvidia: RTX 4070 12GB, RTX 1050Ti 4GB, GT630 2GB, GTS8800, 2xNV28/GF4 Ti4200, NV18/MX440, NV34B/FX5500
Intel:    UHD620, HD610, Atom x5-Z8350, Atom N270 w/ GMA950
ARM:   Mali-T830 MP2, Mali-G76 MP5, Mali-G71
QualComm:   2x Adreno 740, Adreno 308
Sony:   GTE CXD8514Q, 2xGS180nm
SiS:      305 rev2.0

I can't recommend all of them though, some of them aren't very good at computing.

*  Flashed the upgrade rom into the HD6950 myself. Broken now (unrelated***)
** And some older AMD IGP stuff.
*** Actually true

**Reply by AryanEmbered** [reply]
we got a real baller here ladies and gentlemen. 

"I can't recommend all of them though, some of them aren't very good at computing."

*Sees Mx440 on the list*
*nods in agreement*

**Reply by nielsijzerman** [reply]
Damn skippy. If baller means hoarder, even more so.
I found a Nvidia 6600LE by Inno3D as well, and yes, it IS the PCIe 256MB GDDR3 DVI/TV edition! I remember tv out... * sigh * 
That heatsink is so cute, it looks like the box the fan was shipped in, made from tinfoil. The fan looks like it might be noisier than my msi x570 chipset fan. 
That reminds me why I keep al that stuff, see if I can make another one of my famous frankenmods. Notebook cpu cooling with heatpipes perhaps, not a lot of real estate left.

Btw: the GTS8800 has cuda cores, but Bing once told me they wouldn't work. I checked techradar for some specs, because of my silly post here, and behold: 8 series and up support cuda computing. 
If drivers allow it, I will have it team up with the GT630, it has 192 cuda cores. Together that's 2.5GB VRAm and 310 cuda cores. I'll take bets up to $5,- on fail/succes running any llm or tts service, and if succesful, double or nothing on predicting T/s within 0.1 distance. Those are good odds, I predict it to run under 1 T/s. Maybe a Mistral fork in a small Q2 will work.

But first: the 2x HD7850 in crossfire, see if they run HIP Blas, or else how they do in Vulkan.
Is crossfire even a good idea for computing? Probably not, in CF they render frames in turn, if I'm not mistaken, and that will cause lag every time they have to wait for the other to finish. Not sure, just thinking out load, I'll check. We don't want false information in the datasets training our AI, see what Bing witheld from me: 128 cores.

**Reply by cb88** [reply]
Oh its that kind of thread 😈
SX (on board Sun SS20 8MB vsimm which is the framebuffer and DAC on a card wierd right? This one is is a generic SIMD accelerator + dac so it is one of the earliest integrated compute coprocessors you might say)
Sun ZX (can do gourad shaded OpenGL 300k polys etc... )
Fujitsu AG10E (Franken Card it has more GPUs one card than you can shake a stick at I think it has total 28MB of ram on it its 3D accelerator is a GLint 300SX, it also has a Number Nine Imagine 128 , Weitek P9100 ,IBM RGB561 RAMDAC and an Analog Device DSP for custom stuff)
RasterFlex HR (fastest Sun 2D card on sbus)
ATI Wonder 24XL ISA
Vega Video7 ISA card (funny name since I also have a Vega FE)
IBM XGA Microchannel
WX4100 (in my NAS for transcoding)
A few laptops with Radeon HD 7340 and  Radeon HD 8330 Lenovo educational laptops very sturdy I use them for whatever.
GeForce2MX400 
GTX650M
RX460
Vega FE
6800M
There are are few more random ones in there too of course and Mobile GPUs. Do PS3 PS4 and PS5 count?

**Comment by cb88** [comment]
6700XT is what I answered but via 6800M they only have minor differences perf is pretty good compared to my vega FE (even though I can't load quite as large a model at once without splitting).

**Comment by Kashouryo** [comment]
6900XT because I am too broke for a 7900XTX

**Reply by AryanEmbered** [reply]
cries in rx 6600

or as I like to call it, RX shitxyShit hundred

**Comment by vire70** [comment]
"as it doesn't work with the official Rocm implementation on windows unlike other gpus"
Are you implying there is an official Rocm implementation of _Kobold_ ? Or you just mean Rocm on Windows? I'm confused. Anyway I have 7900 XTX on Linux mint. I use linux just as an AI platform to run SD, LLM's, etc. Windows I have tried it in the past but Rocm support still seemed spotty. Plus you have to use really outdated pro drivers to get Rocm installed, which cuts into gaming support.

**Reply by cb88** [reply]
There is no ROCm on Windows... only HIP SDK (which people refer to sometimes as ROCm on windows) HIP SDK is not the same stack as ROCm its an implementation of HIP API on top of the windows driver so of course there are some differences. 

AMD's testing and support of the windows HIP SDK is even spottier than ROCm on Linux with only a handuful of supported GPUS (and thier consumer counterparts except for instance the GRE gpus are not supported at all AFAIK)... 
AMD Radeon PRO W7900 
AMD Radeon PRO W7900
AMD Radeon PRO W7800
AMD Radeon PRO W7700 
AMD Radeon PRO W6800 

Anything other than those is technically unsupported also the W6600 (same as the RX6600 doesn't have HIP SDK support only HIP runtime support so its even spottier).

**Reply by AryanEmbered** [reply]
isn't this fucking insane? Those people should shave their heads and do a japanese spin apology for committing such a grave sin.
